dojo/tools/cobalt_api/parser.py

```diff
@@ -5,20 +5,26 @@
 from dojo.tools.cobalt_api.importer import CobaltApiImporter
 
 
+SCAN_COBALTIO_API = 'Cobalt.io API Import'
+
+
 class CobaltApiParser(object):
     """
     Import from Cobalt.io API /findings
     """
 
     def get_scan_types(self):
-        return ["Cobalt.io API Import"]
+        return [SCAN_COBALTIO_API]
 
     def get_label_for_scan_types(self, scan_type):
-        return "Cobalt.io API Import"
```

Comment:
if you change the scan_type value you probably going to have to write a data migration to convert existing entries?

Reply:
oh wait you're extracting string constants it seems nvm